test/test-handlers.lisp

```diff
@@ -323,7 +323,7 @@ and see what's happening.")
   (let* ((path (get-parameter "path"))
          (file-info (and path
                          (find path *tmp-test-files*
-                               :key 'second :test 'equal))))
+                               :key 'first :test (lambda (a b) (equal a (namestring b)))))))
```

Comment:
is there a reason to not use #'string-equal? I think the types should be perfectly well-defined at this point, and it would be an error in logic elsewhere if that were not the case.